src/pre_commit_vauxoo/cfg/.pre-commit-config.yaml

```diff
@@ -50,7 +50,13 @@ repos:
           - --color
           - --config=.eslintrc.json
           # - --fix
-  - repo: https://github.com/Vauxoo/pre-commit-vauxoo-hooks
-    rev: v0.1.0
+  - repo: local
     hooks:
       - id: vx-check-deactivate
+        name: deactivate.jinja syntax check.
+        description: Validate deactivate.jinja files' syntax.
+        entry: vx-check-deactivate
+        args: []
+        language: python
+        types: [ jinja ]
```

Comment:
I just noticed this extra spaces

I'm curious about this

O could be stripped?

Reply:
They can be stripped, just did it.